onnx/test/shape_inference_test.py

```diff
@@ -100,7 +100,7 @@ def _assert_inferred(
         inferred_model = self._inferred(graph, **kwargs)
         inferred_vis = list(inferred_model.graph.value_info)
         vis = list(sorted(vis, key=lambda x: x.name))
-        inferred_vis = list(sorted(inferred_vis, key=lambda x: x.name))
+        inferred_vis = list(sorted(inferred_vis, key=lambda x: x.name))  # type: ignore
```

Comment:
May I understand which kind of error does it ignore?

Reply:
I would still suggest adding the error code in square brackets to narrow the ignore for clarity.